package.json

```diff
@@ -40,6 +40,7 @@
         "webpack-cli": "^3.3.9"
     },
     "dependencies": {
+        "clipboard": "^2.0.8",
```

Comment:
Hey, appreciate this is the lib we use on the jump page, but was curious how large it is. Turns out it's about ~10KB, and there may be much much smaller libs around.

Given our target browser range will be mobile and likely quite new, worth exploring some alternatives?

https://bundlephobia.com/result?p=clipboard@2.0.8

Not sure what the tradeoffs / edge cases might be.

Reply:
Ahh didn't take a look at the size impact - gee, ok thats relatively big - certainly will look at an alt.

Reply:
Swapped this out for clipboard-copy which has a much smaller footprint - this also behaves the same way as the clipboard lib in a sense as to how it implements the copy operation, however in a way that provides less additional utility and configuration (which we don't require for our needs).